qt/scientific_interfaces/ISISReflectometry/Reduction/PolarizationCorrections.h

```diff
@@ -28,12 +28,31 @@ polarizationCorrectionTypeFromString(std::string const &correctionType) {
     throw std::runtime_error("Unexpected polarization correction type.");
 }
 
+inline std::string
+PolarizationCorrectionTypeToString(PolarizationCorrectionType correctionType) {
+  if (correctionType == PolarizationCorrectionType::None)
```

Comment:
Given that `Reduction/ReductionType.h` uses a switch for enums, should similar pieces of code using both switch or both elif? I don't think it's a necessary thing to change, just wondering if there was a conscious decision behind this?

Reply:
Good spot. These should use `switch` as well, then the compiler will warn if one of the enum values is not handled :+1:

Reply:
Same in FloodCorrections.h